wgengine/magicsock/magicsock.go

```diff
@@ -3027,3 +3027,17 @@ func getPeerMTUsProbedMetric(mtu tstun.WireMTU) *clientmetric.Metric {
 	mm, _ := metricRecvDiscoPeerMTUProbesByMTU.LoadOrInit(key, func() *clientmetric.Metric { return clientmetric.NewCounter(key) })
 	return mm
 }
+
+// GetLastNetcheckReport returns the last netcheck report.
```

Comment:
"...returns the last netcheck report, running a new one if a recent one does not exist."